server/Cargo.toml

```
@@ -3,28 +3,28 @@ strip = true
 
 [workspace.dependencies]
 # dependencies used in graphql crates:
-actix-web = { version = "4.0.1", default-features = false, features = [
+actix-web = { version = "4.7.0", default-features = false, features = [
   "macros",
   "rustls",
 ] }
 actix-multipart = "0.6.1"
-anymap = "0.12"
+anymap = "0.12.1"
 async-graphql = { version = "3.0.35", features = ["dataloader", "chrono"] }
 async-graphql-actix-web = "3.0.35"
-async-trait = "0.1.30"
-chrono = { version = "0.4.23", features = ["serde"] }
+async-trait = "0.1.80"
+chrono = { version = "0.4.38", features = ["serde"] }
 reqwest = { version = "0.12", features = [
   "json",
   "rustls-tls",
   "multipart",
   "blocking",
 ], default-features = false }
-serde = "1.0.126"
-serde_json = "1.0.66"
+serde = "1.0.203"
```

```suggestion
serde = "1.0"
```
Can't we just do this, and always have the latest 1.0 version?

---

Oh yes can!